config/core/configmaps/network.yaml

```diff
@@ -111,8 +111,3 @@ data:
     # 3. Redirected: The Knative ingress will send a 302 redirect for all
     # http connections, asking the clients to use HTTPS
     httpProtocol: "Enabled"
-
```

Comment:
Is this flag included in last release? Is this move a breaking change as operators already set this flag to "true" in this configmap?

Reply:
Yes, the flag is included in the last release and the operators who already set the flag to "enabled" may need to change `config-features` to keep the setting.

Reply:
Sounds like rolling this out will change the behavior of the clusters with this flag set to "enabled". Do we care about this?

Reply:
Basically, I don't think we need to care about that, because this feature is in the alpha stage.
(surely it is good to keep compatibility with the previous version, but keeping such compatibility may cause complexity)

WDYT?

Reply:
I don't have strong options. Just curious.